### ISSUE TITLE
MethodError in maximise 

### ISSUE THREAD

**Issue by mforets** [JuliaIntervals/IntervalOptimisation.jl#39]
```julia
julia> using IntervalOptimisation, IntervalArithmetic

julia>  f(x) = 3x^2 - 1
f (generic function with 1 method)

julia> dom = Interval(2..6)
[2, 6]

julia>  minimise(f, dom)
([11, 11.0054], Interval{Float64}[[2, 2.00089]])

julia> maximise(f, dom)
ERROR: MethodError: no method matching minimise(::getfield(IntervalOptimisation, Symbol("##5#6")){typeof(f)}, ::Interval{Float64}, ::Type{HeapedV
ector}, ::Float64)
Closest candidates are:
  minimise(::Any, ::T; structure, tol) where T at /Users/forets/.julia/dev/IntervalOptimisation/src/optimise.jl:21
Stacktrace:
 [1] #maximise#4(::Type, ::Float64, ::Function, ::typeof(f), ::Interval{Float64}) at /Users/forets/.julia/dev/IntervalOptimisation/src/optimise.j
l:73
 [2] maximise(::Function, ::Interval{Float64}) at /Users/forets/.julia/dev/IntervalOptimisation/src/optimise.jl:73
 [3] top-level scope at none:0
```

**Comment by dpsanders** [comment]
cc @yashcodes

**Comment by dpsanders** [comment]
Fixed by #40
